widget/embedded/src/hooks/useWidgetEvents/useWidgetEvents.ts

```diff
@@ -0,0 +1,7 @@
+import { eventEmitter } from '../../services/eventEmitter';
+
+export function useWidgetEvents() {
```

Comment:
Please specify return type of this function explicitly and also export it from the embedded package root. (otherwise dApp developer doesn't have access to the final exported type)

Reply:
I don't get why dapp developer should have access to the return type of this hook.